flutter(user-feedback): document `lastEventId` (#9677)
getsentry/sentry-docs: docs/platforms/flutter/user-feedback/index.mdx

The user feedback API allows you to collect user feedback while utilizing your own UI. You can use the same programming language you have in your app to send user feedback. In this case, the SDK creates the HTTP request so you don't have to deal with posting data via HTTP.

> Sentry pairs the feedback with the original event, giving you additional insight into issues. Sentry needs the `eventId` to be able to associate the user feedback to the corresponding event. For example, to get the `eventId`, you can use {<PlatformLink to="/configuration/options/#before-send"><PlatformIdentifier name="before-send" /></PlatformLink>} the return value of the method capturing an event. You can also access the id of the latest sent event via `Sentry.lastEventId`.

Reviewer: ```suggestion
Sentry pairs the feedback with the original event, giving you additional insight into issues. Sentry needs the `eventId` to be able to associate the user feedback to the corresponding event. There are several ways to get the `eventId`:
- use {<PlatformLink to="/configuration/options/#before-send"><PlatformIdentifier name="before-send" /></PlatformLink>} 
- use the return value of any method capturing an event. 
- use `Sentry.lastEventId` to get the ID of the last event sent.
```
Not sure if you meant to remove "or" or if it was supposed to be the return value of `beforeSend`... 

I think it'd be easier to read if we list the ways users can get the event id. Please feel free to edit for accuracy and add any additional details.

Author: yup removing the or was not intentional, thx for noticing
